design/design-doc-smaller-providers.md

```diff
@@ -0,0 +1,429 @@
+# Break Up Large Providers by Service
+
+* Owner: Nic Cope (@negz)
+* Reviewers: Hasan Türken (@turkenh), Alper Ulucinar (@ulucinar)
```

Comment:
My rationale here is:

* Hasan is a Crossplane maintainer (who has a lot of experience working on providers)
* Alper is an `upjet` maintainer, and also maintains the three biggest Crossplane providers.